librz/analysis/fcn.c

```diff
@@ -1720,9 +1726,10 @@ RZ_API char *rz_analysis_function_get_json(RzAnalysisFunction *function) {
 	for (i = 0; i < argc; i++) {
 		pj_o(pj);
 		const char *arg_name = rz_type_func_args_name(a->typedb, function->name, i);
-		const char *arg_type = rz_type_func_args_type(a->typedb, function->name, i);
+		RzType *arg_type = rz_type_func_args_type(a->typedb, function->name, i);
+		const char *arg_type_str = rz_type_as_string(a->typedb, arg_type);
```

Comment:
This `char *` should not be `const` and it is leaked. I also see a lot of other places where it is like this.